resources/js/components/FormField.vue

```diff
@@ -137,6 +167,15 @@ export default {
     },
   },
 
+  mounted() {
+    var diff = this.field.minRows - this.fieldsWithValues.length;
+    if (this.canAdd() || Object.keys(this.fieldsWithValues).length === 0 || diff > 0) {
+      for (var i = 0; i < diff; i++) {
```

Comment:
var -> let

Reply:
Var ei meeldi?

Reply:
Var töötab hackylt võrreldes const/let-iga https://hackernoon.com/why-you-shouldnt-use-var-anymore-f109a58b9b70